site/data/dynamo.ts

```diff
@@ -516,6 +562,20 @@ export const removeConsequenceFromDisruption = async (index: number, disruptionI
     );
 };
 
+export const removeSocialMediaPostFromDisruption = async (index: number, disruptionId: string, id: string) => {
+    logger.info(`Updating socialMediaPost ${index} in disruption (${disruptionId}) in DynamoDB table...`);
```

Comment:
Comment needs to be updated to indicate that the social media post is deleted and not updated

Reply:
done